sys/random/fortuna/fortuna.h

```diff
@@ -63,13 +69,17 @@ extern "C" {
 #define FORTUNA_SEED_SIZE           (64U)
 #endif
 
+#if IS_USED(MODULE_FORTUNA_RESEED) || DOXYGEN
 /**
  * @brief Reseed interval in us. After this interval, the PRNG must be
  *        reseeded. Per section 9.5.5, the recommended value is 100ms. Set to
  *        zero to disable this security feature.
+ *
+ * @note Requires `fortuna_reseed` module.
  */
-#ifndef FORTUNA_RESEED_INTERVAL
-#define FORTUNA_RESEED_INTERVAL     (0)
+#ifndef FORTUNA_RESEED_INTERVAL_MS
+#define FORTUNA_RESEED_INTERVAL_MS  0
```

Comment:
Now that we have the pseudomodule as the extra switch for this feature, I'd say let's set the recommended value of 100 ms as default.

Reply:
Hmm seems I bodged the rebase